### ISSUE TITLE
More configuration options for `html-self-closing`

### ISSUE THREAD

**Issue by jrmajor** [sveltejs/eslint-plugin-svelte#837]
### Description

I would like to be able to configure `html-self-closing` so that it aways produces most compact, yet correct tags. That would mean preferring `void` > `self-closing` > `separate closing tag`, but only within spec limitations.

I was able to accomplish that for HTML/Svelte with `{ void: 'never', normal: 'never' }` settings (example output: `<img><div></div><Component/>`), but now it changes SVG tags from `<path/>` to `<path></path>`.

**Comment by Shyam-Chen** [comment]
```sh
$ npx svelte-migrate self-closing-tags
```

**Comment by jrmajor** [comment]
@Shyam-Chen How is that related? Both `<svg><path/></svg>` and `<svg><path></path></svg>` are valid in Svetle 5 and migrate button in Svelte 5 REPL doesn't do anything to them.
